policyengine_us/variables/gov/states/wi/tax/income/credits/refundable/wi_homestead_property_tax.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class wi_homestead_property_tax(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Wisconsin homestead credit property tax amount"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.revenue.wi.gov/TaxForms2021/2021-ScheduleH.pdf#page=2"
+        "https://www.revenue.wi.gov/TaxForms2022/2022-ScheduleH.pdf#page=2"
+    )
+    defined_for = StateCode.WI
+
+    def formula(tax_unit, period, parameters):
+        ptax_owner = add(tax_unit, period, ["real_estate_taxes"])
+        rent = add(tax_unit, period, ["rent"])
```

Comment:
Should we add this? https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/household/expense/utilities/heating_cooling_expense.py

Reply:
I don't see how this would help: this issue for WI is whether or not a person's rent includes heat or not.